Implementation_Report_3e/index.html

```diff
@@ -563,11 +565,11 @@ <h2>2. Coding Independent Code Points (CICP) from ITU-R H.273</h2>
                         <h3>CICP tests</h3>
                         <table>
                             <tr class="header">
-                                <th><a href="https://wpt.fyi/results/png/apng?label=experimental&label=master&aligned">Live Results</a></th>
-                                <th>Chrome 128</th>
-                                <th>Edge 127</th>
-                                <th>Firefox 126</th>
-                                <th>Safari TP 196</th>
+                                <th><a href="https://wpt.fyi/results/png/?label=experimental&label=master&aligned">Live Results</a></th>

```

Comment:
I confirmed this change was intentional. Just marking that I saw it and we're good.